src/main/scala/modbat/dsl/DetExc.scala

```diff
@@ -0,0 +1,10 @@
+package modbat.dsl
+
+import scala.util.matching.Regex
+
+final case class DetExc(excName: Regex, target: State, fullName: String, line: Int)
```

Comment:
Did the best I can with respect to naming by trying to extract context from the source.

Should be trivial to rename if needed.

Reply:
The previous renaming `NonDetExc` was the appropriate one; please name this new class `NonDetExc` again.